CHANGES.md

```diff
@@ -19,6 +19,9 @@
   consider a type declaration recursive if the type appeared inside an attribute
   payload (#299, @NathanReb)
 
+- Append the last valid AST to the error in case of located exception when
+  `embed_errors` is true (#315, @panglesd)
+
```

Comment:
Oh, this is on me: I should have created an empty `unreleased` section right after the last release! On your current branch there's no unreleased section and you've added this to the section of the last release. Would you mind rebasing over main and moving this entry to the `unreleased` section?

And just two things about the entry itself: the user doesn't know what our internal reference `embed_errors` is. So instead of saying "when `embed_errors` is true", you could either formulate it in terms of flags "when `-embed-errors` or `-as-ppx` is set" or in terms of situation "when embedding errors".

And the second thing is that it might be nice to give the context that you're talking about the driver, e.g.: 
```
- Driver: append the last valid AST to the error in case of located exception when (...)
```